### PR TITLE
[PAY-955][PAY-983][PAY-992] Add UI for unsupported collections and other fixes

### PR DESCRIPTION
**Pull request by sddioulde**
### Description

Add web and native mobile UI for unsupported collections.

Also fix small premium content label bug on native mobile. (not super related but wanted to get this change in also)

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

local web dapp and mobile app vs stage

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

existing

### CODE REVIEW

File: packages/web/src/components/data-entry/DropdownInput.module.css [hunk 1]
```diff
@@ -230,3 +233,11 @@
   line-height: 28px;
   height: 32px;
 }
+
+.select .footer:global(.ant-select-item-option) {
+  cursor: default;
+  margin: 0 -8px;
+  padding: 0px !important;
```
Comment (sddioulde) [inline packages/web/src/components/data-entry/DropdownInput.module.css:240]: smell here, but antd styling is weird. don't want to spend too much time on this at this particular time

File: packages/mobile/src/screens/edit-track-screen/screens/ListSelectionScreen.tsx [hunk 2]
```diff
@@ -44,6 +45,7 @@ const useStyles = makeStyles(({ spacing, typography }) => ({
     justifyContent: 'space-between'
   },
   content: { flex: 1 },
+  noFlex: { flex: undefined },
```
Comment (dharit-tan) [inline packages/mobile/src/screens/edit-track-screen/screens/ListSelectionScreen.tsx:48]: Not quite following why no flex is needed when there's a footer, but I trust.

File: packages/mobile/src/screens/edit-track-screen/screens/RemixSettingsScreen.tsx [hunk 3]
```diff
@@ -233,20 +218,22 @@ export const RemixSettingsScreen = () => {
...
-                <Text>{messages.changeAvailbilitySuffix}</Text>
-              </View>
-            </View>
-          )}
+          {isPremium ? (
+            <HelpCallout
+              style={styles.changeAvailability}
+              content={
+                <View style={styles.changeAvailabilityText}>
+                  <Text>{messages.changeAvailbilityPrefix}</Text>
```
Comment (dharit-tan) [inline packages/mobile/src/screens/edit-track-screen/screens/RemixSettingsScreen.tsx:226]: This seems to indicate that availability needs to be changed only when it's not public, but I don't see logic for that.

File: packages/web/src/components/track/PremiumContentLabel.tsx [hunk 4]
```diff
@@ -29,21 +29,7 @@ export const PremiumContentLabel = ({
     return null
   }
 
-  if (isOwner) {
```
Comment (dharit-tan) [inline packages/web/src/components/track/PremiumContentLabel.tsx:32]: Are we no longer showing the labels if `isOwner` or do they appear somewhere else?

**Review by dharit-tan — APPROVED** [review]

**Comment by dharit-tan** [comment]
Need to run prettier!

File: packages/mobile/src/screens/edit-track-screen/screens/ListSelectionScreen.tsx (hunk 2 above)
Comment (sddioulde) [reply packages/mobile/src/screens/edit-track-screen/screens/ListSelectionScreen.tsx:48]: good question. Without this line, the footer gets stuck to the bottom of the screen even if there is only one or a few nft collections in the list. we want the footer to follow right after the list.

File: packages/web/src/components/track/PremiumContentLabel.tsx (hunk 4 above)
Comment (sddioulde) [reply packages/web/src/components/track/PremiumContentLabel.tsx:32]: we are. the logic is refactored and that path would hit the below block

File: packages/mobile/src/screens/edit-track-screen/screens/RemixSettingsScreen.tsx (hunk 3 above)
Comment (sddioulde) [reply packages/mobile/src/screens/edit-track-screen/screens/RemixSettingsScreen.tsx:226]: this shows info that says that we are only able to mark as remix if the track is not premium

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sd-unsupported

**Pull request merged**